R/zzz.R

```diff
@@ -1,4 +1,7 @@
+#' @importFrom backports import
 .onLoad <- function(libname, pkgname) {
+  backports::import(pkgname, "isFALSE")
```

Comment:
I had limited knowledge for this line of code and need help to review this file.

Reply:
We use the function `isFALSE` at one point in the package. It's from `base`, but is only available from 3.4 onward. The `backports` provides functions provided by a later version of R for packages in older version of R. 

We could reimplement it, but `backports` is quite a lightweight package (only dependency is `utils`) and maintained by the R Core Team. My preference is to not try to mimic behavior, but rather rely on the work of the R Core Team to provide these newer functions for older versions that we want to support.